docs/en/jpql-with-kotlin-jdsl/expressions.md

```diff
@@ -48,12 +48,15 @@ avg(path(FullTimeEmployee::annualSalary)(EmployeeSalary::value)).`as`(BigDecimal
 
 Use the following functions to build arithmetic operations:
 
+* abs
```

Comment:
Could `ABS` be included in the operation? I was wondering if it could be a function.